.github/workflows/publish.yaml

```diff
@@ -0,0 +1,34 @@
+name: Publish
+
+on:
+  push:
+    branches:
+      - main
```

Comment:
This will run on merge into `main`, but will only publish if `./.changeset/*.md` are present.